### DISCUSSION TITLE
Was müssen wir tun um ein MVP fertigzubekommen?

### DISCUSSION THREAD

**Discussion by evbc** [evbc/phase1#35] [General]
## 👋 Welcome!
Diese Diskussion soll uns helfen, ein besseres Verständnis dafür zu entwickeln, was es braucht, um ein Minimum Viable Product (MVP) erfolgreich zu entwickeln und vorzeigbar zu machen. Wir werden uns auf die Herausforderungen konzentrieren, die es bei der Entwicklung eines MVPs gibt, und auf Lösungsansätze, die dabei helfen können, diese Herausforderungen zu meistern.

Diese Diskussion soll interaktiv sein und alle Teilnehmer einbeziehen, indem sie die Gelegenheit bietet, ihre eigenen Erfahrungen, Herausforderungen und Lösungsansätze zu teilen. Unser Ziel ist es, am Ende der Diskussion mit neuen Perspektiven und praktischen Empfehlungen ausgestattet zu sein, die uns bei der Entwicklung unserer eigenen MVPs unterstützen können.

**Comment by Th4tM0nk3y** [comment]
Wir müssen wissen, welche Funktionen wir abdecken müssen, d.h. was für Container bzw. Anwendungen benötigt werden, um die Arbeit zu erledigen

**Reply by JohannesBal** [reply]
Eine Datenbank, ein Frontend und eine API. Dürfte für den Fall ja eigentlich reichen oder noch weitere Vorschläge?

**Reply by evbc** [reply]
Wie wärs noch mit einem Setup für einen Cluster?

**Reply by evbc** [reply]
Datenbank und Frontend PoCs stehen per PR bereit.

**Reply by JohannesBal** [reply]
@evbc  PR? Wofür steht das denn? 
Für diesen Fall könnte man einen kind Cluster verwenden.

**Reply by evbc** [reply]
Pull Request

**Reply by evbc** [reply]
Ist Ok. Mach ein Ticket auf. Wir brauchen dafür eine Anleitung

**Comment by Iryn1234** [comment]
Da dieses Produkt an Endkunden geliefert werden soll, müssen wir auch über Security-Aspekte nachdenken. Dazu zählt unter anderem die Verschlüsselung und das Client-Zertifikat

**Reply by evbc** [reply]
Für einen MVP?!?

**Reply by evbc** [reply]
Was für ein Radius?

**Reply by Iryn1234** [reply]
Laut Wiki ist MVP "A minimum viable product (MVP) is a version of a product with just enough features to be usable by early customers who can then provide feedback for future product development (https://en.wikipedia.org/wiki/New_product_development)"
D.h. es wird den endkunden vorgestellt. Aber gut,man kann den Begriff unterschiedlich anwenden  :)

**Reply by evbc** [reply]
Eben :)

**Reply by yk94x** [reply]
Radius an wieviele Tester geliefert wird, ich denke das macht aber aktuell keinen unterschied?

**Comment by evbc** [comment]
Wo sind hier eigentlich die Smileys :)

**Comment by evbc** [comment]
Faszinierend! Meine PRs werden applied, obwohl ich da übelste Mängel eingebaut habe...

**Reply by Th4tM0nk3y** [reply]
Als ich eben drüber geschaut habe, sind mir keine gravierende Fehler aufgefallen :sweat_smile:

**Reply by evbc** [reply]
@yk94x Warum hast Du den Adminer PR ohne Approvals gemerged?

**Reply by evbc** [reply]
![Bildschirm­foto 2023-02-10 um 14 52 19](https://user-images.githubusercontent.com/123601944/218108567-304757a3-6658-41b0-b7e2-755ff38ae23f.png)

Kein gravierender Mangel?

**Reply by JohannesBal** [reply]
Passwörter sollten so nicht offen verwendet werden!

**Reply by yk94x** [reply]
ist mir nicht direkt aufgefallen

**Comment by mahadyalahow** [comment]
Pull request

**Comment by evbc** [comment]
Mir scheint wir haben alles beisammen. Können wir die Diskussion abschliessen? Haben wir alle Issues für einen MVP?

**Reply by Th4tM0nk3y** [reply]
Ich denke wir haben soweit alles

**Comment by evbc** [comment]
Na dann...
